src/runtimes/node/runtime.ts

```diff
@@ -6,6 +6,7 @@ import { Network } from './net_info';
 import xhrAuth from 'isomorphic/auth/xhr_auth';
 import { AuthTransports } from 'core/auth/auth_transports';
 import xhrTimeline from 'isomorphic/timeline/xhr_timeline';
+import * as crypto from 'crypto';
```

Comment:
Could you only import the function we need (`randomInt`)?

Reply:
Done.